.github/workflows/create_release.yml

```diff
@@ -32,15 +35,22 @@ jobs:
             ~/.m2/repository
             ~/.ivy2/cache
             ~/.cache/coursier
+            ~/.sbt/launchers
+            ~/.sbt/boot
+            ~/.sbt/preloaded
```

Comment:
?

Reply:
We cannot add `~/.sbt` as a whole because of our custom private_repository.sbt file...
So I list all existing directories to cache.